sopel/config/core_section.py

```diff
@@ -152,33 +189,72 @@ class CoreSection(StaticSection):
     Used by the URL callbacks feature to call plugins when links are posted in
     chat; see the :func:`sopel.module.url` decorator.
 
-    The default value allows ``http``, ``https``, and ``ftp``.
+    This is equivalent to the default values:
+
+    .. code-block:: ini
+
+        auto_url_schemes =
+            http
+            https
+            ftp
+
     """
 
     bind_host = ValidatedAttribute('bind_host')
     """Bind the connection to a specific IP.
 
     :default: ``0.0.0.0`` (all interfaces)
+
+    This is equivalent to the default value:
+
+    .. code-block:: ini
+
+        bind_host = 0.0.0.0
+
     """
 
     ca_certs = FilenameAttribute('ca_certs', default=_find_certs())
     """The path to the CA certs ``.pem`` file.
 
-    If not specified, Sopel will try to find the certificate trust store itself.
+    Example:
+
+    .. code-block:: ini
+
+        ca_certs = /etc/ssl/certs/ca-certificates.crt
+
+    If not specified, Sopel will try to find the certificate trust store
+    itself.
     """
 
     channels = ListAttribute('channels')
     """List of channels for the bot to join when it connects.
 
     If a channel key needs to be provided, separate it from the channel name
-    with a space, e.g. ``"#channel password"``.
+    with a space:
+
+    .. code-block:: ini
+
+        channels =
+            "#channel"
+            "#logs"
+            &rare_prefix_channel
+            "#private password"
+
+    .. important::
+
+        When you edit the config file manually, make sure to escape each line
+        starting with a ``#`` with double quotes (as shown in the example
+        above), otherwise Python will ignore them, thinking they are comments.
```

Comment:
I tried to improve this, with mixed results.

```suggestion
        If you edit the config file manually, make sure to wrap each line
        starting with a ``#`` in double quotes, as shown in the example above.
        An unquoted ``#`` denotes a comment, which will be ignored by Sopel's
        configuration parser.
```

Would a link to [the ConfigParser syntax docs](https://docs.python.org/3/library/configparser.html#supported-ini-file-structure) be overkill here?

Reply:
No that's a good idea. I'll add this suggestion and I'll try not to forget about the link!